client/app.jsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import Home from './pages/home';
+import Navbar from './components/navbar';
+import CreateEvent from './pages/create-event';
+import Events from './pages/events';
+import Event from './pages/event';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 export default class App extends React.Component {
```

Comment:
Could you change this to a hook-based component? (Not urgent, but it is very easy.)